front/src/reducers/osrdconf/helpers.ts

```diff
@@ -105,3 +108,47 @@ export const insertVia = (
   updatedVias.splice(adjustedIndex, 0, newVia);
   return updatedVias;
 };
+
+export const insertViaV2 = (
```

Comment:
This function is only called if we insert a via from the map, maybe we should specify it in the title of the function ?
```suggestion
export const insertViaFromMap = (
```

Reply:
Done